x-pack/plugins/data_visualizer/public/application/common/components/top_values/top_values.tsx

```diff
@@ -49,7 +49,10 @@ export const TopValues: FC<Props> = ({ stats, fieldFormat, barColor, compressed
   } = stats;
   const progressBarMax = isTopValuesSampled === true ? topValuesSampleSize : count;
   return (
-    <EuiFlexItem data-test-subj={'dataVisualizerFieldDataTopValues'}>
+    <EuiFlexItem
+      data-test-subj={'dataVisualizerFieldDataTopValues'}
+      className={'dataVisualizerPanelWrapper'}
```

Comment:
Is there a reason why you aren't adding a panel around the expanded row content for `boolean` type fields:

![image](https://user-images.githubusercontent.com/7405507/132861694-650ab3ac-91d1-450b-8819-490b6b3e9e75.png)

Reply:
Fixed here https://github.com/elastic/kibana/pull/107184/commits/6a7b1bad6fd5f97a2a361738c9d2a98268bf0dd9